osquery/tables/system/windows/windows_search.cpp

```diff
@@ -0,0 +1,399 @@
+// osquery/tables/system/windows/windows_search.cpp
+
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <windows.h>
```

Comment:
The osquery [coding standard](https://llvm.org/docs/CodingStandards.html) recommends grouping headers by their purpose; you can try leaving osquery headers at the end (I've manually reordered them, I didn't test this out)

// Windows headers
#include <windows.h>
#include <atlbase.h>
#include <atldbcli.h>
#include <comutil.h>
#include <searchapi.h>

// Standard Library headers
#include <codecvt>
#include <sstream>
#include <strsafe.h>
#include <vector>

// osquery headers
#include <osquery/core/core.h>
#include <osquery/core/tables.h>
#include <osquery/logger/logger.h>
#include <osquery/utils/conversions/windows/strings.h>
#include <osquery/utils/conversions/windows/windows_time.h>
#include <osquery/utils/conversions/join.h>
#include <osquery/utils/scope_guard.h>

Reply:
done